sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -525,9 +524,8 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
    * @param options - Options that can be set while sending the request.
    * @returns New lock token expiry date and time in UTC format.
    */
-  async renewLock(lockToken: string, options?: SendManagementRequestOptions): Promise<Date> {
+  async renewLock(lockToken: string, options: SendManagementRequestOptions = {}): Promise<Date> {
```

Comment:
No default needed (I'm going to stop commenting on these now!)

Reply:
Thanks, I'll address all these at once.

Reply:
Searched for `options?:` and fixed all of them as you suggested for the OperationOptionsBase.